makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
```

Comment:
You also never require the binary to be set. I understand that we only want to do something if spec is set, but for Makeflow we should also have the binary. There is a comment on this in the umbrella wrapper. Assuming we need both, then the logic should check for both.